<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ALPS-PSO4L" name="com.test.TestProject" tests="4" failures="0" timestamp="22 Sep 2016 06:31:28 GMT" time="1.000" errors="1">
  <testcase name="testProjectGet" time="0.069" classname="com.test.TestProject">
    <error type="java.lang.AssertionError" message="Getting Projects details failed.... expected [DISP-000219] but found [Rest Project]">
      <![CDATA[java.lang.AssertionError: Getting Projects details failed.... expected [DISP-000219] but found [Rest Project]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at com.test.TestProject.testProjectGet(TestProject.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at execution.Execution.execute(Execution.java:55)
	at execution.ExecuteSuite.main(ExecuteSuite.java:10)
]]>
    </error>
  </testcase> <!-- testProjectGet -->
  <testcase name="testProjectPost" time="0.476" classname="com.test.TestProject"/>
  <testcase name="testProjectDelete" time="0.353" classname="com.test.TestProject"/>
  <testcase name="testProjectPut" time="0.102" classname="com.test.TestProject"/>
</testsuite> <!-- com.test.TestProject -->
